mealie/schema/recipe/recipe_ingredient.py

```diff
@@ -74,7 +74,7 @@ def validate_quantity(cls, value, values) -> NoneFloat:
         empty string.
         """
         if isinstance(value, float):
-            return value
+            return round(value, 3)
```

Comment:
I added this because in the case of 1/3 I was getting a value of `0.333333` on the frontend/backend and it seemed like a waste of space.

Do you see anything wrong with rounding here? Seems to operator the same to me as far as I've tested

Reply:
Nope that looks good to me! That's pretty much the same thing I did in the parser anyway, so it makes sense here too